src/ray/rpc/retryable_grpc_client.h

```diff
@@ -201,7 +206,8 @@ class RetryableGrpcClient : public std::enable_shared_from_this<RetryableGrpcCli
   // to prevent OOM.
   const uint64_t max_pending_requests_bytes_;
   const uint64_t check_channel_status_interval_milliseconds_;
-  const uint64_t server_unavailable_timeout_seconds_;
+  const int32_t server_unavailable_timeout_seconds_;
+  const bool server_call_unavailable_timeout_immediately_;
```

Comment:
Seems these two variables are mutually exclusive, if `server_call_unavailable_timeout_immediately_` is set we don't care about `server_unavailable_timeout_seconds_` anymore. I thought we want to call immediately only for the very first retry and then honor `server_unavailable_timeout_seconds_` afterwards. Did I miss anything here?